health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,341 @@
+    if (layer?.remove) {
+      map.removeLayer(layer);
+    }
+  });
+};
+
+const ViewElement = ({ type, label }) => {
+  const renderIcon = () => {
+    switch (type) {
+      case "Village":
+        return (
+          <svg xmlns="http://www.w3.org/2000/svg" width="27" height="39" viewBox="0 0 27 39" fill="none">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)